mitmproxy/tools/console/grideditor/editors.py

```diff
@@ -54,16 +54,24 @@ def set_data(self, vals, flow):
 
 
 class RequestFormEditor(base.FocusEditor):
-    title = "Edit URL-encoded Form"
```

Comment:
Minor: I'd like to have two seperate editors here, which both accept either format as input but only output the respective format. This makes it possible to quickly change the form type from within mitmproxy.